### PR TITLE
build pypi package for seerep gRPC api

### PR DESCRIPTION
**Pull request by jarkenau**
This PR includes:

- Providing the seerep gRPC api and message  via pypi
- Protobuf python files are located in `seerep.pb`
- Flatbuffers python files are located in `seerep.fb`

Remaining Work:
- [x] Ensure consistency for including the build python classes in the dev-container and from pypi. Related Issue: https://github.com/agri-gaia/seerep/issues/237
- [x] Add README on how to use the pypi packages.

Note for review:
- The C++ files can be ignored, the protobuf namespace changed from `seerep` to `seerep::pb`

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
         "matepek.vscode-catch2-test-adapter",
         "davidanson.vscode-markdownlint",
         "cschlosser.doxdocgen",
-        "cheshirekow.cmake-format"
+        "cheshirekow.cmake-format",
+		"tamasfe.even-better-toml"
```
Comment (Mark-Niemeyer) [inline .devcontainer/devcontainer.json:20]: wrong indent.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -102,76 +102,43 @@ jobs:
...
+      - name: Build pypi packages
         run: |
-          cd $GITHUB_WORKSPACE/src/python-package
-          python3 -m build --sdist
+          python3 -m build $GITHUB_WORKSPACE/src
 
-      - name: publish
-        if: github.ref_type == 'tag'
+      - name: Publish packages to pypi
+        #if: github.ref_type == 'tag'
```
Comment (Mark-Niemeyer) [inline .github/workflows/main.yml:136]: Add this if before merge

File: pyproject.toml [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+
+[project]
+name = "seerep_grpc"
+description = "Package for the SEEREP gRPC API"
+readme = "README.md"
+license = {text = "BSD 3-Clause License"}
+requires-python = ">=3.8"
+keywords = ["SEEREP, gRPC, Protocol Buffers", "Flatbuffers"]
+authors = [
+    {name = "Mark Niemeyer", email = "mark.niemayer@dfki.de"}
```
Comment (Mark-Niemeyer) [inline pyproject.toml:13]: wrong email. Typo in last name

**Review by Mark-Niemeyer — CHANGES_REQUESTED** [review]

**Review by Mark-Niemeyer — DISMISSED** [review]

File: examples/python/gRPC/images/gRPC_fb_addBoundingBox.py [hunk 4]
```diff
@@ -7,8 +7,8 @@
 
 import flatbuffers
 import numpy as np
-from fb import BoundingBox2DLabeledWithCategory, BoundingBoxes2DLabeledStamped, Image
```
Comment (Mark-Niemeyer) [inline examples/python/gRPC/images/gRPC_fb_addBoundingBox.py:10]: I guess the deleted `BoundingBox2DLabeledWithCategory` is still needed and should be added again

**Review by Mark-Niemeyer — CHANGES_REQUESTED** [review]

File: examples/python/gRPC/images/gRPC_fb_addBoundingBox.py (hunk 4 above)
Comment (jarkenau) [reply examples/python/gRPC/images/gRPC_fb_addBoundingBox.py:10]: Nope, not bounding box import needed here, it's handled by the `util_fb`

**Comment by jarkenau** [comment]
Now in #250, it was easier to cherry-pick the commits rather than resolving all the conflicts.

**Pull request closed without merging**